sdk/src/beta9/abstractions/endpoint.py

```diff
@@ -98,6 +98,8 @@ def multiply(**inputs):
         ```
     """
 
+    concurrent_requests = 1
```

Comment:
I think i was wondering if we even needed to initialize for endpoints. Seems unnecessary

Reply:
If we don't initialize then when we are deploying an endpoint we get an attribute not found error because we pass `self.concurrent_requests` into the runner's `init`.

Reply:
doesnt it get initialize here with a default?
https://github.com/beam-cloud/beta9/pull/613/files#diff-330a31c80611143ecc706c48c89d09bd4f73e343ae53bcfa4a606048da630b46R102

Reply:
It won't ever reach that line because [this line](https://github.com/beam-cloud/beta9/pull/613/files#diff-7925030b37f47ede4bed18d7a45561a55ce011de54bbb098e0437c2bfe0a915fR140) will cause a missing attribute error. 

For ASGI apps it won't cause a missing attribute error because it has a default value in the ASGI `init` method. However, we don't want this to be an option on the Endpoint decorator, so there is no default set in the `init` because its not in the `init` method.  

By setting this default in the Endpoint, we guarantee that `concurrent_requests` is an attribute that can be passed into the runner's init method. When it is set in the ASGI `init`, it overrides this default value.

Reply:
can this line be 
```
concurrent_requests=1,
```

instead of 
```
concurrent_requests=self.concurrent_requests,
```

since endpoints doesnt care too much about it?

Reply:
We can't set that because then L256 `self.concurrent_requests = concurrent_requests` will have no effect. The value of `concurrent_requests` passed into the ASGI `init` method will just be overwritten with `1` every time  because ASGI init -> Endpoint init -> Runner init. 

Basically, by using `self.concurrent_requests`, we are able to override that value in the ASGI `init`, but it will still always be `1` for endpoint.

Reply:
hmm ok